Updated Table documentation (#1357)
bigbinary/neeto-ui: stories/Components/TableStoriesDocs/TableDocs.mdx

`import { Table } from "@bigbinary/neetoui";`

> We use Ant Design's `Table` component under the hood. For extra customization, refer [AntD Table](https://ant.design/components/table/#API).

Reviewer: This line is actually needed. Otherwise we are not pointing out anywhere about this fact, but still will be linking to the AntD docs.

const columnData = [
  {
    title: "ID",
    dataIndex: "id",
    key: "id",
    width: 75,
    sorter: (a, b) => a.id - b.id,
  },
  // Other column defenitions
  ]

const rowData = [
  {

>     id: "1",

Reviewer: Is `id` supposed to be a `string` or a `number`? If it's a string, then would the sorting condition that you've written above generate the desired outcome?

Author: Technically it should work because of the type-coercion. But this can cause confusion, So will be changing it.

    // Data for other columns
  },
  {
    id: "2",
    // Data for other columns
  }

  // Other rows

];

<Table
  columnData = {columnData}
  rowData = {rowData}

>   />

Reviewer: Will we be able to successfully render a neetoUI table by just passing these two props alone? If not, then let's also point out the essential props.

];
```

Keys for a column definition:

> - `title` : The title of a column. Accepts a react Node.

Reviewer: ```suggestion
- `title` : The title of a column. Accepts a React Node(JSX) or a string.
```

Author: One thing I've missed here is that `title` also accepts a method that returns a ReactNode. I'll be adding that too.

- `dataIndex` : The unique key over which, the row data is mapped over for the particular column. Accepts a string or an array of strings.
- `key` : Unique identifier of a column. Accepts a string.

> - `render` : Defines a render method of that column. Accepts a method that returns a react Node.

Reviewer: ```suggestion
- `render` : Defines how data should be rendered within the column. Accepts a callback that returns a React Node.
```
